packages/client/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@overturebio-stack/lectern-client",
```

Comment:
Need to rename to `@overture-stack/lectern-client`. Once published, we should publish a deprecation notice with a link to the new package to the [original package location](https://www.npmjs.com/package/@overturebio-stack/lectern-client)